### PR TITLE
mention CORS in Core > Assets

### PR DESCRIPTION
**Pull request by cvan**
r? @dmarcos @ngokevin

### CODE REVIEW

File: src/docs/core/assets.md [hunk 1]
```diff
@@ -15,3 +15,17 @@ To keep everything in one place and to later help A-Frame decide what to block o
...
+
+Per [Wikipedia](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing):
+
+> Cross-origin resource sharing (CORS) is a mechanism that allows restricted resources on a web page to be requested from another domain outside the domain from which the resource originated.
+
+> The reason we have to set these properties is that, by default, the modern browser checks whether you're allowed to request a resource from a different domain than the one you're running on. When you use Three.js to load a model or a texture, it uses an  XMLHTTPRequest  object to access that resource. Your browser will check for the availability of the correct headers, and since you're requesting a resource from your local system, which doesn't provide the correct headers, an error will oc- cur. Even, though with this recipe, you can circumvent this restriction, it is better to always test with a local web server, since that will most closely resemble how your users will access it online. For more information on CORS, refer to http://www.w3.org/TR/cors/
```
Comment (cvan) [inline src/docs/core/assets.md]: typos

**Comment by dmarcos** [comment]
It looks good to me but I wonder whether this is necessary. CORS is a constraint of the Web in general and affects to any development workflow regardless of the library or framework of choice. I'm not sure if it  should be covered in our docs.

**Comment by ngokevin** [comment]
Thanks, adapted it to https://github.com/aframevr/aframe/commit/43b6d78b4df388ab079c5217e211a2639d9f2ff0

**Comment by cvan** [comment]
many thanks!

**Pull request closed without merging**